ietfdata/mailarchive.py

```diff
@@ -97,7 +118,23 @@ def message_from_archive_url(self, archive_url:str) -> Message:
     def messages(self) -> Iterator[Message]:
```

Comment:
Does the return type need to change - it looks like this now returns a Tuple?

Reply:
Fixed.